Aiolos/Aiolos/Sources/PanelConfiguration.swift

```diff
@@ -74,6 +74,7 @@ public extension Panel {
         public var supportedModes: Set<Mode>
         public var gestureResizingMode: GestureResizingMode
         public var appearance: Appearance
+        public var shouldHideHandleSpace: Bool = true
```

Comment:
- we don't specify default values here directly, for consistency reasons this should be kept the same
```suggestion
        public var shouldHideHandleSpace: Bool
```

Reply:
I think we should move this property to the `Appearance` sub-struct - we already have 2 resizeHandle-related properties there, so I would make this an enum with associated values

    enum ResizeHandleAppearance {
          case hidden
          case visible(foregroundColor: UIColor, backgroundColor: UIColor)
    }